tests/integration/pydecred/test_rpc.py

```diff
@@ -35,3 +35,19 @@ def test_rpc(config):
 
     blockchainInfo = rpcClient.getBlockchainInfo()
     assert isinstance(blockchainInfo, rpc.GetBlockChainInfoResult)
+
+    validateAddress = rpcClient.validateAddress("DsUxwT6Kbiur6Nps9q3uGEpJCvrhcxX2nii")
+    assert isinstance(validateAddress, rpc.ValidateAddressChainResult)
```

Comment:
Probably want to test the negative path here too. At least with one nonsense address. Could do an incorrect network as well for good measure.